all-components/navigation.css

```diff
@@ -0,0 +1,82 @@
+@import url('./utilities.css');
+
+
+/*main navigation styling*/
+.main-nav {
+    padding: 0 1.5rem;
+    background: var(--PRIMARY-COLOR);
+    width: 100%;
+    height: 4.5rem;
+    box-shadow: 0 0 10px 5px var(--GREY);
+
```

Comment:
as its 10px I think using 1rem would be better

Reply:
Mt root font size if 16px, that's why I used 10px.